doc/book/admin/start_stop_instance.rst

```diff
@@ -1,214 +1,335 @@
+    .. code-block:: console
 
-To check the instance file for syntax errors prior to starting ``my_app``
-instance, say:
+        $ tt status my_app
+        INSTANCE     STATUS          PID
+        my_app       NOT RUNNING
 
-.. code-block:: console
+*   ``tt restart`` -- restart the instance:
 
-    $ tarantoolctl check my_app
+    .. code-block:: console
 
-To enable ``my_app`` instance for auto-load during system startup, say:
+        $ tt restart my_app -y
+           • The Instance my_app (PID = 729) has been terminated.
+           • Starting an instance [my_app]...
 
-.. code-block:: console
+    .. note::
 
-    $ systemctl enable tarantool@my_app
+        The ``-y`` option responds "yes" to confirmation prompt automatically.
 
-To stop a running ``my_app`` instance, say:
+*   ``tt stop`` -- stop the instance:
 
-.. code-block:: console
+    .. code-block:: console
 
-    $ tarantoolctl stop my_app
-    $ # - OR -
-    $ systemctl stop tarantool@my_app
+        $ tt stop my_app
+           • The Instance my_app (PID = 639) has been terminated.
 
-To restart (i.e. stop and start) a running ``my_app`` instance, say:
+*   ``tt clean`` -- remove instance artifacts: logs, snapshots, and other files.
 
-.. code-block:: console
+    .. code-block:: console
 
-    $ tarantoolctl restart my_app
-    $ # - OR -
-    $ systemctl restart tarantool@my_app
+        $ tt clean my_app -f
+           • List of files to delete:
 
-.. _admin-start_stop_instance-running_locally:
+           • /var/log/tarantool/my_app.log
+           • /var/lib/tarantool/my_app/00000000000000000000.snap
+           • /var/lib/tarantool/my_app/00000000000000000000.xlog
 
---------------------------------------------------------------------------------
-Running Tarantool locally
---------------------------------------------------------------------------------
+    .. note::
 
-Sometimes you may need to run a Tarantool instance locally, e.g. for test
-purposes. Let's configure a local instance, then start and monitor it with
-``tarantoolctl``.
+        The ``-f`` option removes the files without confirmation.
 
-First, we create a sandbox directory on the user's path:
+.. _admin-start_stop_instance-multi-instance:
 
-.. code-block:: console
+Multi-instance applications
+---------------------------
 
-    $ mkdir ~/tarantool_test
+Tarantool applications can include multiple instances that run different code.
+A typical example is a cluster application that includes router and storage
+instances. The ``tt`` utility enables managing such applications.
+With a single ``tt`` call, you can:
 
-... and set default ``tarantoolctl`` configuration in
-``$HOME/.config/tarantool/tarantool``. Let the file contents be:
+*   start an application on multiple instances
+*   check the status of application instances
+*   connect to a specific instance of an application
+*   stop a specific instance of an application or all its instances
 
-.. code-block:: lua
+Application layout
```

Comment:
Future task: we need to think about a new place for this section as it not directly related to administration. Ideally, this section should contain only the `Managing multi-instance applications` content.